data/server.js

```diff
@@ -0,0 +1,74 @@
+
+
+const express = require('express'); 
+const router = express.Router(); 
+const server = express(); 
+
+const actionDB = require('./helpers/actionModel')
+const projectsDB = require('./helpers/projectModel'); 
+
+
+
+// ACTION req handlers
+
+
+router.get('/', (req, res) => {
```

Comment:
if you're not going to route outside of the file you have to write out the url path
so if its get would be get '/actions', (req, res)